content/docs/studio/self-hosting/configuration/index.md

```diff
@@ -0,0 +1,59 @@
+# Configuration
+
+Studio uses a standardized, unified YAML configuration file, often referred to
+as `values.yaml` in the documentation.
+
+## Updating the configuration
+
+To update the configuration and apply the changes, follow the instructions
+below:
+
+<admon type="info">
+
+If you've deployed Studio with the AMI, you'll need to SSH to the EC2 instance
+before continuing.
+
+</admon>
+
+1. Update Studio's `values.yaml` config file
+
+**Example: Updating the hostname**
+
+Merge the existing `values.yaml` file with the following contents:
+
+```yaml
+global:
+  host: studio.company2.com
+```
+
+3. Reload Studio
+
+```cli
+$ helm upgrade --wait studio iterative/studio --namespace studio -f values.yaml
+```
+
+## More configuration options
+
+The previous chapter provided a basic example of updating the Studio
+configuration. In most cases, you'll likely need to do more configuration to set
+up Studio to your needs.
+
+Studio integrates with several Git forges (also referred to as SCM providers),
```

Comment:
SCM providers - should be subsection or subpage imo?

Reply:
Yes, good idea. We can make it a subsection